.github/test-infra/ci-iac-aws/loki/main.tf

```diff
@@ -0,0 +1,153 @@
+# test tf
```

Comment:
Just going to comment this here but applies to all the terraform folders. These should be able to be consolidated into one with a loop on a couple of values to keep the repo DRY